invenio_vocabularies/contrib/names/datastreams.py

```diff
@@ -82,9 +83,11 @@ def _process_lambda_file(self, fileobj):
                 # Lambda file is ordered by last modified date
                 last_modified_str = row["last_modified"]
                 try:
-                    last_modified_date = arrow.get(last_modified_str, date_format)
-                except arrow.parser.ParserError:
-                    last_modified_date = arrow.get(
+                    last_modified_date = datetime.strptime(
+                        last_modified_str, date_format
+                    )
+                except Exception:
+                    last_modified_date = datetime.strptime(
```

Comment:
what is the reason for not using arrow package?

Reply:
We were already using arrow for only a very small number of cases that could be nearly fully handled using only Python standard library methods from `datetime` so it seemed nicer to get rid of a dependency here